[feat] Update Audit Trail docs (#819)
grafana/k6-docs: src/data/markdown/docs/03 cloud/03 Manage/06 Audit Trail.md

You can scan the audit trail for abnormalities and even check whether the IP addresses correlate to what you have 
registered for your users.

> ## Viewing Results
> - Highlighted in green is the period filter which allows you to view results within a specific time frame.
> - Highlighted in orange are columns that can sort the results.

Reviewer: ```suggestion
## Viewing Results

- To view results within a specified time frame, use the **period** filter (highlighted in green).
- To change how results are sorted, use the columns (highlighted in orange).
```

Author: Thank you! 🥳
